src/utils/post_utils.js

```diff
@@ -383,3 +383,26 @@ export function combineSystemPosts(postsIds = [], posts = {}, channelId) {
 
     return {postsForChannel, nextPosts};
 }
+
+export function isPostCommentMention({post, currentUser, threadRepliedToByCurrentUser, rootPost}) {
+    let commentsNotifyLevel = 'never';
+    let isCommentMention = false;
+    let threadCreatedByCurrentUser = false;
+
+    if (rootPost && rootPost.user_id === currentUser.id) {
+        threadCreatedByCurrentUser = true;
+    }
+    if (currentUser.notify_props && currentUser.notify_props.comments) {
+        commentsNotifyLevel = currentUser.notify_props.comments;
+    }
+
+    const notCurrentUser = post.user_id !== currentUser.id || (post.props && post.props.from_webhook);
+    if (notCurrentUser) {
+        if (commentsNotifyLevel === 'any' && (threadCreatedByCurrentUser || threadRepliedToByCurrentUser)) {
+            isCommentMention = true;
+        } else if (commentsNotifyLevel === 'root' && threadCreatedByCurrentUser) {
```

Comment:
I think we can use the constants at `constants/preferences`:
```
COMMENTS_ANY: 'any',
COMMENTS_ROOT: 'root',
COMMENTS_NEVER: 'never',
```

Reply:
Definitely. Made the change.